styles/theme/_uswds-theme-spacing.scss

```diff
@@ -0,0 +1,92 @@
+/*
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+========================================
+========================================
+========================================
+----------------------------------------
+USWDS 2.13.1
```

Comment:
I think I mentioned this on another PR but this settings file is for USWDS v2, but I think the package.json has USWDS v3. Can you make these match the same major version?

Reply:
Update, good catch.